product_docs/docs/pem/8/pem_query_tool.mdx

```diff
@@ -137,19 +137,19 @@ On successful generation of `Explain` plan, it will create three tabs/panels und
 
 ### Graphical tab
 
-Hover over an icon on the `Graphical` tab to review information about that item; a popup window will display information about the selected object. For information on JIT statistics, triggers and a summary, hover over the icon on top-right corner; a similar popup window will be displayed when appropriate.
+Hover over an icon on the `Graphical` tab to review information about that item; a popup window displays information about the selected object. For information on JIT statistics, triggers, and a summary, hover over the icon on top-right corner; a similar popup window is displayed when appropriate.
 
 Please note that `EXPLAIN VERBOSE` cannot be displayed graphically.
 
-Use the download button on top left corner of the `Explain` canvas to download the plan as an SVG file. Please note that `Download as SVG` feature is not supported on Internet Explorer.
+Use the download button on the top left corner of the `Explain` canvas to download the plan as an SVG file. Please note that `Download as SVG` feature is not supported on Internet Explorer.
 
 ![Query Tool - Explain tab - Graphical plan tab](images/query_output_explain_details.png)
 
 The query plan that accompanies the `Explain analyze` is available on the `Data Output` tab.
 
 ### Analysis tab
 
-The `Analysis` tab shows the plan details in table format, it generates a format similar to the format available at `explain.depsez.com`. Each row of the table represents the data for a `Explain Plan Node`. The output may contain the node information, exclusive timing, inclusive timing, actual vs. planned rows, actual rows, planned rows, or loops. When you select a row, the child rows of that selected row are marked with an orange dot.
+The `Analysis` tab shows the plan details in table format, generating a format similar to the one available at `explain.depsez.com`. Each row of the table represents the data for a `Explain Plan Node`. The output may contain the node information, exclusive timing, inclusive timing, actual vs. planned rows, actual rows, planned rows, or loops. When you select a row, the child rows of that selected row are marked with an orange dot.
```

Comment:
Tab names are bold. Use "an" before vowel...I don't think you need to tag Explain Plan Node at all...I think the word "node" can probably be l/c. Don't use may--use might or can. Don't abbreviate versus. Should the website be a link? Tightened up last sentence.

The **Analysis** tab shows the plan details in table format, generating a format similar to the one available at `explain.depsez.com`. Each row of the table represents the data for an Explain Plan node. The output can contain the node information, exclusive timing, inclusive timing, actual versus planned rows, actual rows, planned rows, or loops. Child rows of the selected row are marked with an orange dot.